test/test-project/package.json

```
@@ -8,7 +8,7 @@
     },
     "dependencies": {
         "@asyncapi/generator": "file:../..",
-        "@asyncapi/html-template": "^0.23.8"
+        "@asyncapi/html-template": "^0.16.0"
```

I think that it should be fixed without `^`

```suggestion
        "@asyncapi/html-template": "0.16.0"
```

---

but bump workflow with bump it anyway